bioblend/galaxy/workflows/__init__.py

```diff
@@ -401,6 +426,13 @@ def invoke_workflow(
         :param use_cached_job: Whether to use cached jobs for the workflow
           invocation.
 
+        :type resource_params: dict
+        :param resource_params: A dictionary containing the resource parameters
+          to be used for this workflow run.
+
+        :type instance: bool
+        :param instance: True if the provided id is Workflow id, as opposed to StoredWorkflow id.
```

Comment:
For consistency with the `instance` parameter of `get_workflows()`:
```suggestion
        :param instance: treat ``workflow_id`` as a Workflow ID if True,
          otherwise treat it as a StoredWorkflow ID (the default). This
          parameter works only on Galaxy 21.05 or later.
```